script/tunnel

```diff
@@ -1,23 +1,20 @@
 #!/bin/bash
-# 
+#
 # Start tunnel for server using ngrok or localtunnel
-# 
+#
 
 DIR=$(dirname "${BASH_SOURCE[0]}")/
 source "${DIR}/utils.sh"
 
 if [ -z ${TUNNEL_PORT} ]; then
   echo "TUNNEL_PORT must be set"
   exit 1
-elif [ -z ${TUNNEL_SUBDOMAIN} ]; then
-  echo "TUNNEL_SUBDOMAIN must be set"
-  exit 1
 fi
 
 if is_installed ngrok; then
-  exec ngrok http "${TUNNEL_PORT}" --subdomain="${TUNNEL_SUBDOMAIN}" --log=stdout
+  exec ngrok http "${TUNNEL_PORT}" --log=stdout
```

Comment:
Q: Can we do a local setup without this? As I understand if we not using the tunnel subdomain a random URL will be generated by ngrok and the same URL needs to be used as DOMAIN in App config and env file. prior to running the script.